Save vector feature to file alg update (#8708)
qgis/QGIS-Documentation: docs/user_manual/processing_algs/qgis/vectorgeneral.rst

     - [string]
     - GDAL layer creation options of the output format.
       Separate individual options with semicolons.
   * - **Action to take on pre-existing file**

>        Default

Reviewer: ```suggestion
```
The default value goes to the Type column

     - ``ACTION_ON_EXISTING_FILE``

>      - [string]
>      - Managing existing features.
>        Create or overwrite file.
>        Create or overwrite layer.
>        Append features to existing layer, but do not create new fileds.
>        Append features to existing layer, and create new fields if needed.  

Reviewer: ```suggestion
     - [enumeration]

       Default: 0
     - How to manage existing features. Valid methods are:

       0 --- Create or overwrite file
       1 --- Create or overwrite layer
       2 --- Append features to existing layer, but do not create new fields
       3 --- Append features to existing layer, and create new fields if needed
```
While shown as string, this option/widget actually provides a set of predefined values exposed as numbers to the alg when run. We use enumeration type here.

     - [string]
     - GDAL layer creation options of the output format.
       Separate individual options with semicolons.
   * - **Action to take on pre-existing file**

     - ``ACTION_ON_EXISTING_FILE``
     - [enumeration]

       Default: 0
     - How to manage existing features. Valid methods are:

       0 --- Create or overwrite file
       1 --- Create or overwrite layer

>        2 --- Append features to existing layer, but do not create new fileds

Reviewer: ```suggestion
       2 --- Append features to existing layer, but do not create new fields
```
